vizro-core/src/vizro/figures/_kpi_cards.py

```diff
@@ -129,6 +135,11 @@ def kpi_card_reference(  # noqa: PLR0913
         >>> vm.Page(title="Page", components=[vm.Figure(figure=kpi_card_reference(...))])
 
     """
+
+    def _get_footer_class(delta: float, reverse_color: bool) -> str:
+        pos_color, neg_color = ("color-neg", "color-pos") if reverse_color else ("color-pos", "color-neg")
+        return pos_color if delta > 0 else neg_color if delta < 0 else ""
+
```

Comment:
This makes the function just a little cleaner, but that's totally up to you. I agree with both versions.
```suggestion

    def _get_footer_class(delta: float, reverse_color: bool) -> str:
        if delta == 0:
            return ""
            
        pos_color, neg_color = ("color-neg", "color-pos") if reverse_color else ("color-pos", "color-neg")
        return pos_color if delta > 0 else neg_color

```